libs/models/schema/ui/buttons/src/lib/slider-button-data.interface.ts

```diff
@@ -0,0 +1,27 @@
+export interface SliderButtonData extends ButtonData
+{
+  text: string;
+
+  bgColour?: string; // Background colour
+  color?: string;
+  borderColor?: string;
+  iconColor?: string;
+  iconBackgroundColor?: string;
+
+  hoverBgColour?: string;
+  hoverColor?: string;
+  hoverBorderColor?: string;
+  hoverIconColor?: string;
+  hoverIconBackgroundColor?: string;
+}
+
+export interface ButtonData 
+{
+  text: string
+  
+  bgColor?: string; // Background colour
+  color?: string; // text color
+  
+  hoverBgColor?: string;
+  hoverColor?: string;
+}
```

Comment:
Move this interface to a separate file and then import it.

Reply:
the buttonData had not been created yet but I have used another interface file and imported the dummy interface